native_client/ctcdecode/ctc_beam_search_decoder.cpp

```diff
@@ -96,24 +96,52 @@ DecoderState::next(const double *probs,
         if (full_beam && log_prob_c + prefix->score < min_cutoff) {
           break;
         }
+        if (prefix->score == -NUM_FLT_INF) {
+          continue;
+        }
+		if (!prefix->is_empty() && prefix->timesteps.empty()) {
```

Comment:
Here and throughout: please use spaces and fix the indentation to match the rest of the file.

Reply:
Oh, I forgot to check that, I will fix it.